controllers/teamsController.js

```diff
@@ -0,0 +1,78 @@
+const { thesisTeams } = require('../config/teams');
+const Team = require('../helpers/team');
+const daysAgo = 7;
```

Comment:
This is mysterious.  I'd love for you to find either a name that explains better what this is about or add a comment to enlighten other devs.